packages/experiments/src/components/Text/Text.tsx

```diff
@@ -17,8 +17,10 @@ export interface ITextProps {
   type?: keyof IFontTypes;
   family?: keyof IFontFamilies;
   size?: keyof IFontSizes;
+  style?: keyof IFontStyles;
   weight?: keyof IFontWeights;
-  color?: keyof IFontColors;
+  color?: keyof IPalette;
```

Comment:
I'd do color?: ISemanticColor | IPalette
without the keyof

Reply:
In that case would the user be expected to pass in an entire IPalette or ISemanticColor definition? We're just expecting them to pass in a string for color, like `<Text color="red" />` .

Reply:
oops you're right, I got confused, definitely have the keyof

Reply:
It doesn't like that either :/ It gives the same type of error message

Reply:
I suppose if we're moving in that direction, I could cast color as the two different keyof types.
`const paletteColor = color as keyof IPalette;`
`const semanticColor = color as keyof ISemanticColors;`